tests/workers/xmodel/test_resnet50.py

```diff
@@ -68,14 +69,27 @@ def get_config():
         )
         return (model, parameters)
 
+    def set_request(self, images):
```

Comment:
Add this logic to check the modelMetadata to `ImageInferenceRequest()` as well. Would this function be needed then?

Reply:
added